### PR TITLE
Don't chdir globally

### PR DESCRIPTION
**Pull request by tagomoris**
Calling Dir.chdir in top level of Rakefile makes us not to add any tasks which should run on project root directory.
IMO, it's better to show working directory per tasks explicitly than doing chdir it globally.

### CODE REVIEW

**Comment by tagomoris** [comment]
I know #43 (and I'm waiting it'll be merged and released), but it seems a bit large for quick merge.
This one is just removing global `Dir.chdir` to add another tasks.

**Comment by zzak** [comment]
Agreed, this is a good first step

**Comment by zzak** [comment]
Could you rebase? <3

**Comment by tagomoris** [comment]
I want to add this change on #47. What's the status of it?

**Comment by zzak** [comment]
@tagomoris merged.

**Comment by tagomoris** [comment]
@zzak done.

**Comment by toch** [comment]
Thanks @tagomoris 
those changes need also to be applied to the Rakefile. Could you also please add them there?

have you checked how it plays with the changes that will be soon introduced by #43 ?

**Comment by zzak** [comment]
@toch I think this patch as a smaller version of Eric's patch that should be easier to swallow, and allow us to move into that direction.

**Comment by tagomoris** [comment]
I pushed a commit to update Rakefile by updated setup.rb

**Comment by tagomoris** [comment]
ping?

File: Rakefile [hunk 1]
```diff
@@ -25,27 +24,31 @@ current_gem = MRuby::Gem.current
 app_version = MRuby::Gem.current.version
 APP_VERSION = (app_version.nil? || app_version.empty?) ? "unknown" : app_version
 
-desc "compile all the binaries"
+desc "compile binary"
```
Comment (zzak) [inline Rakefile]: this still compiles "all" binaries, right? why change the comment

File: Rakefile [hunk 2]
```diff
@@ -63,21 +66,25 @@ namespace :test do
...
+        end
       end
     end
   end
 end
 
 desc "run all tests"
 Rake::Task['test'].clear
-task :test => ['test:bintest', 'test:mtest']
+task :test => ["test:mtest", "test:bintest"]
```
Comment (zzak) [inline Rakefile]: why change the order and quotations?

File: Rakefile [hunk 3]
```diff
@@ -133,8 +140,7 @@ Rake.application.tasks.each do |task|
     old_task = Rake.application.instance_variable_get('@tasks').delete(task.name)
     desc old_task.full_comment
     task old_task.name => old_task.prerequisites do
-      abort("Not running in docker, you should type \"docker-compose run <task>\".") \
-        unless is_in_a_docker_container?
+      abort("Not running in docker, you should type \"docker-compose run <task>\".")         unless is_in_a_docker_container?
       old_task.invoke
```
Comment (zzak) [inline Rakefile:147]: Can you put this back on two lines?

**Comment by zzak** [comment]
`setup.rb` LGTM, please make the changes I mentioned to `Rakefile` and I will merge

File: Rakefile (hunk 1 above)
Comment (tagomoris) [reply Rakefile]: I previously worked with Rakefile generated by mruby-cli v0.0.4.
Now I fixed and pushed it.

File: Rakefile (hunk 2 above)
Comment (tagomoris) [reply Rakefile]: Pushed commits without such unneeded changes.

File: Rakefile (hunk 3 above)
Comment (tagomoris) [reply Rakefile:147]: ditto.

**Comment by tagomoris** [comment]
@zzak Thank you to notify me about it. I pushed fixed commits, and another fix for `release` task.
It works well for my project.

**Pull request merged**